python_ta/reporters/color_reporter.py

```diff
@@ -50,3 +50,10 @@ def _colourify(cls, colour_class: str, text: str) -> str:
         space_count = len(text) - len(new_text)
         new_text = new_text.replace(" ", cls._SPACE)
         return (space_count * cls._SPACE) + colour + new_text + cls._COLOURING["reset"]
+
+
+from pylint.lint import PyLinter
```

Comment:
Group this import statement with the others at the top of this file. (Same comment applies to all reporter files.)